src/utils/Utils.ts

```diff
@@ -0,0 +1,13 @@
+const Utils = {
+  encodeNormalizedName(uri: string) {
```

Comment:
I am surprised this works at all. What happens when you specify functions like this in an object? Is the code shortcut for something like:
```ts
const Utils = {
   encodeNormalizedName: function(uri: string) {
      ...
   }
}
```
?

In any case, in JS / TS it's preferred not to encapsulate utils functions like this into an object because if you do, you cannot import only a single utility, you always have to import the whole batch. In a sense, modules in JS / TS work quite close to what classes do in OOP languages - it's a good enough encapsulation.

The better approach here is just to convert it to a list of plain named functions and export those directly, e.g.:
```ts
export function encodeNormalizedName(....) { ... }
```

When importing, you can do either
```ts
import { encodeNormalizedName } from `./utils'
```
or
```ts
import * as Utils from './utils'
```